cli/ballerina-launcher/src/main/java/org/ballerinalang/launcher/LauncherUtils.java

```diff
@@ -89,7 +89,9 @@ public static void runProgram(Path sourceRootPath, Path sourcePath, String funct
         ProgramFile programFile;
         String srcPathStr = sourcePath.toString();
         Path fullPath = sourceRootPath.resolve(sourcePath);
-        loadConfigurations(sourceRootPath, runtimeParams, configFilePath, observeFlag);
+        // Set the source root path relative to the source path i.e. set the parent directory of the source path
+        System.setProperty("ballerina.source.root", fullPath.getParent().toString());
```

Comment:
Is it safe to keep this sysprop as is ?

Reply:
This system property is added to give users the ability to retrieve the source root path via the Config API

Reply:
👍